test/test_forms.py

```diff
@@ -0,0 +1,112 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.forms import (
+    CarModelSearchForm,
+    ManufacturerNameSearchForm,
+    DriverLicenseUpdateForm,
+    DriverUsernameSearchForm,
+    DriverCreationForm
+)
+from taxi.models import Manufacturer
+
+
+class TestCarForms(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="new_driver",
+            password="root1234",
+        )
+        self.client.force_login(self.user)
+
+        self.manufacturer = Manufacturer.objects.create(
+            name="Test manufacture", country="USA"
+        )
+
+    def test_car_search_form_with_arg(self):
+        form_data = {
+            "model": "M5",
+        }
+        form = CarModelSearchForm(data=form_data)
+        self.assertTrue(form.is_valid())
+
+    def test_car_search_form_without_arg(self):
+        form_data = {
+            "model": "",
+        }
+        form = CarModelSearchForm(data=form_data)
+        self.assertTrue(form.is_valid())
+
+
+class TestManufacturerForms(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="new_driver",
+            password="root1234",
+        )
+        self.client.force_login(self.user)
+
+        self.manufacturer = Manufacturer.objects.create(
+            name="Test manufacture", country="USA"
+        )
+
+    def test_manufacturer_search_form_with_arg(self):
+        form_data = {
+            "name": "Test manufacture",
+        }
+        form = ManufacturerNameSearchForm(data=form_data)
+        self.assertTrue(form.is_valid())
+
+    def test_manufacturer_search_form_without_arg(self):
+        form_data = {
+            "name": "",
+        }
+        form = ManufacturerNameSearchForm(data=form_data)
+        self.assertTrue(form.is_valid())
+
+
+class TestDriverForms(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            username="new_driver",
+            password="root1234",
+        )
+        self.client.force_login(self.user)
+
+        self.manufacturer = Manufacturer.objects.create(
+            name="Test manufacture", country="USA"
+        )
+
+    def test_driver_form(self):
```

Comment:
Make name of you tests functions more sense (For example `test_driver_creation_form_is_valid_with_valid_data`)